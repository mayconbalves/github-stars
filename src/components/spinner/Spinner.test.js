import { render } from '@testing-library/react'
import Spinner from './'

describe('<Spinner />', () => {
  it('should render Spinner', () => {
    const { container, getByTestId } = render(<Spinner />)
    expect(getByTestId('spinner')).not.toBeNull()
    expect(container.firstChild).toMatchSnapshot()
  })
})
