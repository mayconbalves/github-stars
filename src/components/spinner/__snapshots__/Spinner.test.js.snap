// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render Spinner 1`] = `
<div
  class="sc-bdfBwQ gZPsER"
  data-testid="spinner"
>
  <div
    class="sc-gsTCUz bwgWST"
  />
</div>
`;
